src/pages/TakeTurnPage.vue

```diff
@@ -93,10 +94,28 @@ const onBodegasGetted = (bodegas: Bodega[]) => {
   bodegaSelect.value.options.length = 0;
   bodegaSelect.value.selected = null;
   bodegaSelect.value.options.push(...bodegas);
-  step.value++;
+
+  if (bodegaSelect.value.options.length == 1) {
+    warehouseCant.value = 1;
```

Comment:
This condition is the same that say 'warehouseCant.value = bodegaSelect.value.options.length"